test/unit/CBot/CBot_test.cpp

```diff
@@ -228,7 +229,7 @@ class CBotUT : public testing::Test
         program->Compile(code, tests);
 
         CBotError error;
-        int cursor1, cursor2;
+        std::size_t cursor1 = 0, cursor2 = 0;
```

Comment:
Few lines below there are comparisons of `cursor1` and `cursor2` >= 0, which is always true in case of std::size_t. Our linter points it out as a warning, but because compiler is set up to treat warnings as errors, it causes tests to fail.
I think it should be safe to remove them now.